### DISCUSSION TITLE
Atropos with React bug, where transform not added for children

### DISCUSSION THREAD

**Discussion by boylerplet** [nolimits4web/atropos#42] [Q&A]
I have tried to use this component in my Project, with many unsuccessful attempts. The Problem I noticed was that when I create my component using the same images as in the demo site, The images do not have transform style/properties added to it. 

For reference you can check the produced version vs. demo version markup using the devtools:

*Produced:*
![image](https://github.com/nolimits4web/atropos/assets/54875908/56c225d4-0111-4225-b082-2cbb34740088)

*On demo( documentation ) site:*
![image](https://github.com/nolimits4web/atropos/assets/54875908/c487061c-e4be-4d6f-a59f-5fec80c07903)

This means that the style added to all children of the `Atropos` Component is not being added.

I think this is a major bug and an issue can be opened for it.

**Comment by nolimits4web** [comment]
Hard to say anything without live example with the issue

**Reply by boylerplet** [reply]
Okay. Here is an MWE (https://codesandbox.io/s/atropos-bug-react-5tq27d) of the bug I am encountering. I have just created the component as per the [documentaion website](https://atroposjs.com/docs/react).

I have tried importing css in three ways 
- Using `index.html`
- Using import statement using relative path in `index.js`
- Using import statement using absolute path in `App.js`

I am not sure If that is the problem though. Absolute positioning in the `img` is not applied.

**Comment by nolimits4web** [comment]
In your codesandbox everything works correctly

**Reply by boylerplet** [reply]
Okay. It was my mistake. Now I see. I did not understand that The classes of absolute should be added for each child of the Atropos component.
